docs/vaults/BalleMaster.md

```diff
@@ -50,3 +51,44 @@ Each user will get his part of rewards from the vault, proportional to his share
+
+No tokens are held in any of the two contracts of the vault, they only pass thru them to reach the destination's platform farm smart contract when the user makes a deposit deposit and the inverse path on withdrawal.
+
+For the creation of a new vault, a new strategy smart contract will be deployed to the blockchain. Then a new entry to vault's registry on BalleMaster smart contract will be created with the address of the tokens involved and the strategy smart contract address. This step will be made by a call to `addVault()`.
+At this point, the vault is created, so it can be tested that deposit and withdraw functions are working properly (`deposit()`, `withdraw()`, `emergencyWithdraw()` and `harvest()`), to ensure that there will be no problems with user's tokens.
+
+The only thing that is not working still is the BALLE rewards distribution to ensure that testing phase previous to launch will not interfere with rewards distribution.
+
+For activation of BALLE reward distribution on the new vault, a transaction from our GNOSIS Management Safe should be made. This is to ensure that no one person alone can change rewards distribution on vaults. The GNOSIS safe is set as owner of our BalleMaster smart contract after deployment on mainnet and all methods involved on reward distribution and balancing shoul be made from it. For this transactions to be made, a determined number of signatures on the GNOSIS Safe will be needed.
+
+### Rewards activation
+
+After the creation and testing of the new vault, it will need an activation from the GNOSIS Management Safe that will call the `activateVaultRewards()` method of the smart contract. With this call, where the `allocPoint` for the vault should be indicated, all needed variables for the vault get initialized and reward distribution starts from the next block onwards.
+
+### Rewards modification
+
+If, at some point, the rewards distribution multiplicator for one vault needs to be changed, it can be made from the GNOSIS Management Safe with a call to `modifyVaultRewards()` method of the smart contract with the new `allocPoint` for the vault.
+
+### Rewards deactivation
+
+In case a vault needs to be retired, it should be deactivated from rewards distribution to not interfere with the active vaults. This will be made from our GNOSIS Management Safe with a call to `deactivateVaultRewards()` method of the smart contract.
+
+## Vault information for frontend
+
+There is some information about the vaults to show in the frontend app that should be obtained here.
+
+The first, is the visibility of the vault itself. The vault shoul be made visible on our frontend on the same moment the activation occurs, and while this can not be determined exactly (depends when the last needed signature on the GNOSIS is made), there should be a method to show the vault automatically when it's activated (reading his status information from the smart contract)
```

Comment:
```suggestion
The first is the visibility of the vault itself. The vault should be made visible on our frontend at the same moment the activation occurs, and while this can not be determined exactly (depends when the last needed signature on the GNOSIS is made), there should be a method to show the vault automatically when it's activated (reading his status information from the smart contract)
```